content/en/docs/KRE/user/40_kais_runner_sdk/_index.md

```diff
@@ -24,10 +117,52 @@ description: >
 
 ## Context
 
+The context object provides a set of utilities as database client or logger that can be used
```

Comment:
could be nice to add "a database client" "a logger"

Reply:
I think i'm gonna delete the sentence about and part of these phrase to avoid redundancy